File: Aula06/exercicios/ex04.cpp
#include <iostream>
#include <string>
using namespace std;

string mensagem(string nome, int idade){
    return "Ola " + nome + ", voce tem " + to_string(idade) + " anos!";
}

int main(){
    string nome;
    int idade;

    cin >> nome >> idade;
    cout << mensagem(nome, idade) << endl;
}

File: Aula06/exercicios/ex04.py
nome = input("Nome: ")
idade = int(input("Idade: "))
print(f"Olá {nome}, você tem {idade} anos!")

# Versão 2
def mensagem(nome, idade):
    return f"Olá {nome}, você tem {idade} anos!"

print(mensagem(nome, idade))